static/js/message_list.js

```diff
@@ -26,7 +26,7 @@ export class MessageList {
         const table_name = opts.table_name;
         this.view = new MessageListView(this, table_name, collapse_messages);
         this.table_name = table_name;
-        this.narrowed = this.table_name === "zfilt";
+        this.narrowed = true; // Making narrowed true for all tables including home
```

Comment:
While this seems to work for the use case of the banner, the "all messages" view isn't a narrow so this should not be changed - this might end up affecting more things than intended. It might be better to just not check for this variable when displaying / hiding the banner

Reply:
> While this seems to work for the use case of the banner, the "all messages" view isn't a narrow so this should not be changed - this might end up affecting more things than intended. It might be better to just not check for this variable when displaying / hiding the banner

Changing narrowed true just for "zfilt" results in the following bug:

https://user-images.githubusercontent.com/96344003/210390678-87bbc4eb-b89a-4a65-9f95-9aba0d011054.mov

Thus, to re-render all messages any time a stream is unmuted, I changed narrowed value to true.

Reply:
Ok. I'll make the changes.